src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewMoney.razor

```diff
@@ -0,0 +1,3 @@
+﻿@using static ViewMoney
```

Comment:
same here, I think we don't need it as ViewMonay doesn't have any static members

Reply:
similar to EditMoney